tests/spec/date_series_ops/test_date_series_ops.py

```diff
@@ -166,6 +166,29 @@ def test_difference_between_dates_in_years(spec_test):
     )
 
 
+def test_difference_between_dates_in_days(spec_test):
+    table_data = {
+        p: """
+              |     d1
+            --+------------
+            1 | 2000-01-01
+            2 | 2000-02-01
+            3 | 1999-12-30
+            5 |
+            """,
+    }
+    spec_test(
+        table_data,
+        (p.d1 - "2000-01-01").days,
+        {
+            1: 0,
+            2: 31,
+            3: -2,
+            5: None,
+        },
+    )
+
+
```

Comment:
Maybe add cases over the end of Feb for leap/non-leap years so the behaviour is super-clear?